lib/api/controllers/securityController.js

```diff
@@ -0,0 +1,61 @@
+var
+  ResponseObject = require('../core/models/responseObject');
+
+module.exports = function SecurityController (kuzzle) {
+
+  /**
+   * Get a specific role according to the given id
+   * @param {RequestObject} requestObject
+   * @returns {Promise}
+   */
+  this.getRole = function (requestObject) {
+    kuzzle.pluginsManager.trigger('data:getRole', requestObject);
+
+    return kuzzle.repositories.role.loadRole(
+      kuzzle.repositories.role.getRoleFromRequestObject(requestObject)
+      )
+      .then(role => {
+        return Promise.resolve(new ResponseObject(requestObject, role || {}));
```

Comment:
Looks like the response does not follow the new responseObject format. We get the role object directly in the result, not in _source:
```
{ status: 200,
  error: null,
  requestId: '6904a370-baa5-11e5-94df-d3e4c4b5fa6d',
  controller: 'security',
  action: 'getRole',
  metadata: { iwant: 'to break free', we: [ 'will', 'rock', 'you' ] },
  state: 'done',
  scope: null,
  result: { indexes: { fakeIndex1: [Object] }, closures: {}, _id: 'test' } }
```